include/highfive/H5Object.hpp

```diff
@@ -104,7 +91,7 @@ class Object {
 class ObjectInfo  {
   public:
     /// \brief Retrieve the address of the object (within its file)
-    haddr_t getAddress() const noexcept;
+    H5_DEPRECATED haddr_t getAddress() const noexcept;
```

Comment:
Could this be enhanced with what is replacing it?  ie: `H5_DEPRECATED("Return type will change to size_t")` or whatever the plan is?